mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/MainActivity.java

```diff
@@ -131,31 +166,44 @@ protected void onCreate(Bundle savedInstanceState) {
     PermissionHelper.checkAndRequestCameraPermissions(this);
 
     // Send loaded 3d render assets as side packets to graph
+
+    // Load a basic GIF for initialization of GIF assets
+    setGIFBitmaps("https://i.giphy.com/media/fDO2Nk0ImzvvW/source.gif");
     prepareDemoAssets();
     AndroidPacketCreator packetCreator = processor.getPacketCreator();
     Map<String, Packet> inputSidePackets = new HashMap<>();
-    inputSidePackets.put(SIDE_PACKET_ASSET_TAG, packetCreator.createString(OBJ_FILE));
-    inputSidePackets.put(SIDE_PACKET_TEXTURE_TAG, packetCreator.createRgbaImageFrame(objTexture));
+    inputSidePackets.put(ROBOT_TEXTURE_TAG, packetCreator.createRgbaImageFrame(robotTexture));
+    inputSidePackets.put(ROBOT_ASSET_TAG, packetCreator.createString(ROBOT_FILE));
+    inputSidePackets.put(DINO_TEXTURE_TAG, packetCreator.createRgbaImageFrame(dinoTexture));
+    inputSidePackets.put(DINO_ASSET_TAG, packetCreator.createString(DINO_FILE));
+    inputSidePackets.put(GIF_ASSET_TAG, packetCreator.createString(GIF_FILE));
     processor.setInputSidePackets(inputSidePackets);
 
     // Add frame listener to PacketManagement system
     mediaPipePacketManager = new MediaPipePacketManager();
     processor.setOnWillAddFrameListener(mediaPipePacketManager);
 
+    // Send device properties to render objects via OpenGL
+    Map<String, Packet> devicePropertiesSidePackets = new HashMap<>();
+    devicePropertiesSidePackets.put(ASPECT_RATIO_SIDE_PACKET_TAG, packetCreator.createFloat32(ASPECT_RATIO));
```

Comment:
nit: This should come from querying the video frame.  Fine to leave for now with a TODO.